crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -418,4 +437,11 @@ mod test {
         let err = result.unwrap_err();
         assert_eq!(err.to_string(), "Failed to convert value = yyyy to Felt252");
     }
+
+    #[test]
+    fn contract_address_from_felt_252() {
+        let felt = Felt252::from(112_233);
+        let addr = contract_address_from_felt252(&felt).unwrap();
+        assert_eq!(stark_felt_to_felt(*addr.0.key()), felt);
+    }
```

Comment:
Does it test anything meaningful?

Reply:
I didn't want to neglect coverage as we manipulate bytes to go from one type to another. In this PR I didn't replace it everywhere, but if it's used in more locations, we may want to ensure that the helper is converting the value correctly.

Reply:
I mean it does not hurt to test it, but I don't think we test anything meaningful here. The only thing the tested function does is struct construction , so we are basically testing blockifier's code here

Reply:
Sorry I didn't get you well. Test was removed due to your suggestion to convert it using blockifier `felt_to_stark_felt`. :+1: